examples/plot_custom_colors.py

```diff
@@ -0,0 +1,71 @@
+"""
+=================================
+Plot RGBA values on brain surface
+=================================
+
```

Comment:
@LauraGwilliams can you add a sentence to explain what this example does?

Reply:
Done, yes!